Bottom: c12c02e3fb7e5772e6c644b094d3e8f5149571d1
Top:    479639fc81d96039e102284bcc56060494a29ef9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 16:03:15 +0100

Just make a black background.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 3c24c60..94f3055 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1998,9 +1998,9 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glGetFloatv (GL_MODELVIEW_MATRIX, (GLfloat *)last_modelview_matrix);
 
   glEnable (GL_STENCIL_TEST);
-  glClearColor (port->offlimits_color.red / 65535.,
-                port->offlimits_color.green / 65535.,
-                port->offlimits_color.blue / 65535.,
+  glClearColor (port->bg_color.red / 65535.,
+                port->bg_color.green / 65535.,
+                port->bg_color.blue / 65535.,
                 1.);
   glStencilMask (~0);
   glClearStencil (0);
@@ -2084,6 +2084,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   /* Drawing operations as masked to areas where the stencil buffer is '0' */
   /* glStencilFunc (GL_GREATER, 1, 1); */           /* Draw only where stencil buffer is 0 */
 
+#if 0
   if (global_view_2d) {
     glBegin (GL_QUADS);
     glVertex3i (0,             0,              0);
@@ -2114,6 +2115,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     }
     glEnd ();
   }
+#endif
 
   ghid_draw_bg_image ();
